build/config/webpack.browseralt.prod.js

```diff
@@ -0,0 +1,14 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+import prodConfig from './webpack.base.snippets.prod';
+import browserConfig from './webpack.browseralt.default';
+
+export default {
+  ...prodConfig,
+  ...browserConfig,
+  plugins: [
+    ...prodConfig.plugins,
+    ...browserConfig.plugins,
+  ],
+};
```

Comment:
Should have a transformer function that creates these dev and prod configurations, instead of having to spread things ourselves into object literals. Dev and prod look the same for every single configuration we've written so far, so it's probably safe to assume a generic transformer would suffice.